src/math/FGFunction.cpp

```diff
@@ -29,6 +29,8 @@ INCLUDES
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%*/
 
 #include <iomanip>
+#include <random>
+#include <chrono>
 
```

Comment:
You must include `<memory>` for the compilation to succeed on Linux.
```suggestion
#include <memory>

```

Reply:
I put the #include <memory> i my last version